src/plugin.tsx

```diff
@@ -80,23 +80,23 @@ export class GlossaryPlugin {
     }
 
     let studentInfo;
-    if (this.context.remoteEndpoint) {
```

Comment:
I just realized that in order for logging to work, the AP is going to need to have access to the remoteEndpoint point. Clue must be doing this for its logging, and it is launched from the portal the same way as the AP. 

So now I think it'd be better to fix the AP instead of changing the glossary plugin.